web/src/AppController.ts

```diff
@@ -126,7 +126,7 @@ class AppController {
     fetch(url)
```

Comment:
where is this function used?

Reply:
It's a function invocation, not a function definition. Is that what you mean?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API